t_http2/test_h2_sticky_scheduler.py

```diff
@@ -0,0 +1,164 @@
+"""Test module for http2 and Sticky Cookie Scheduler."""
+import http
+
+from framework import tester
+from helpers import dmesg
+from helpers.response_parser import parse_response
+
+
+class H2StickySchedulerTestCase(tester.TempestaTest):
+    """Sticky Cookie H2 test case."""
+
+    clients = [
+        {
+            'id': 'curl-init',
+            'type': 'external',
+            'binary': 'curl',
+            'ssl': True,
+            'cmd_args': '-Ikf -v --http2 https://${tempesta_ip}:8765/ -H "Host: good.com"',  # noqa:E501
+        },
+    ]
+
+    tempesta = {
+        'config': """
+            listen ${tempesta_ip}:8765 proto=h2;
+
+            srv_group default {
+                server ${server_ip}:8000;
+            }
+
+            vhost v_good {
+                proxy_pass default;
+                sticky {
+                    sticky_sessions;
+                    cookie enforce;
+                    secret "f00)9eR59*_/22";
+                }
+            }
+            tls_match_any_server_name;
+            tls_certificate ${tempesta_workdir}/tempesta.crt;
+            tls_certificate_key ${tempesta_workdir}/tempesta.key;
+            cache 0;
+            cache_fulfill * *;
+            block_action attack reply;
+            http_chain {
+                host == "bad.com"	-> block;
+                host == "good.com" -> v_good;
+            }
+        """,
+    }
+
+    backends = [
```

Comment:
Wouldn't it be sufficient to just use `deproxy` backend here?

Reply:
I have no answer. Perhaps, deproxy did not work with h2 at the moment when I started this task.